pages/common/rustup.md

```diff
@@ -0,0 +1,24 @@
+# rustup
+
+> Rust toolchain installer.
+> Install, manage, and update Rust toolchains.
+
+- Install the nightly toolchain for your system:
+
+`rustup install nightly`
```

Comment:
What does `rustup install` install by default ? We should probably start with the simplest form of the command and then build on to more complex versions.

Reply:
`rustup install` prints and error by default.  You have to specify a toolchain.

Reply:
Got it. Thanks for the clarification.